app/controllers/incomeSources/add/AddIncomeSourceStartDateController.scala

```diff
@@ -91,7 +91,7 @@ class AddIncomeSourceStartDateController @Inject()(authenticate: AuthenticationP
 
     val messagesPrefix = incomeSourceType.startDateMessagesPrefix
 
-    withSessionData(JourneyType(Add, incomeSourceType)) { _ =>
+    withSessionData(JourneyType(Add, incomeSourceType), initialPage = {if (incomeSourceType != SelfEmployment) true else false}) { _ =>
```

Comment:
Should fix:
You can simplify by passing the param as incomeSourceType != SelfEmployment, if/else is redundant